### DISCUSSION TITLE
Date is earliest message, not most recent in inbox list view

### DISCUSSION THREAD

**Discussion by WilliamAGH** [protz/thunderbird-conversations#2020] [Q&A] [answered]
Hello,

It appears the date/time stamp in the inbox list view is of the earliest message in the thread/conversation, not the most recent that just arrived.

Is this the intended functionality? The inbox is already being sorted by descending order (most recent on top). 

Thanks everyone

**Comment by Standard8** [comment, marked as answer]
Hi,

This is already on file as #160. The change would need to be on Thunderbird's side as the message list is deeply integrated into Thunderbird. There is already this bug filed about it: https://bugzilla.mozilla.org/show_bug.cgi?id=604033

Although we have an issue tracking it, it isn't something that I think Conversations would do, as Conversations is primarily about changing the message display to include the whole conversation.
